coops-ui/src/mixins/resource-lookup-mixin.ts

```diff
@@ -59,8 +59,11 @@ export default class ResourceLookupMixin extends Vue {
       }
 
       if (configSection.canadianResident) {
-        if (directors.filter(x => (x.deliveryAddress.addressCountry !== 'CA' ||
-        (x.mailingAddress && x.mailingAddress.addressRegion !== 'BC'))).length === directors.length) {
+        const count = directors.length
+        const notCanadian = directors.filter(x => (x.deliveryAddress.addressCountry !== 'CA' ||
+        (x.mailingAddress && x.mailingAddress.addressCountry !== 'CA'))).length
+
+        if (notCanadian / count > 0.5) {
```

Comment:
Is there any chance we'll get here with 0 directors?

Reply:
I see that a check was added for the directors object to exist and for its length to not be null/zero. Therefore, count should never be zero and this code is now safe.